build/constants.yaml

```diff
@@ -346,15 +346,23 @@
     directory: src/startupscriptgenerator/src/common/consts
 - name: hugo-constants
   constants:
-    version: 0.96.0
+    version: 0.112.5
     platform-name: hugo
     installed-hugo-versions-dir: /opt/hugo
     installation-url-format: https://github.com/gohugoio/hugo/releases/download/v#VERSION#/#TAR_FILE#
     tar-file-name-format: hugo_extended_#VERSION#_Linux-64bit.tar.gz
-    toml-file-name: config.toml
-    yaml-file-name: config.yaml
-    yml-file-name: config.yml
-    json-file-name: config.json
+    toml-file-name: 
```

Comment:
nit: since these values are now lists, can we pluralize the name of the variables? _i.e._, `toml-file-name` ==> `toml-file-names`